src/pages/schema-edit/parse-schema.js

```diff
@@ -31,6 +31,21 @@ function processSections(sections) {
     });
 }
 
+function processSelected(children) {
+    var hasSelected = children.find((option) => option.selected);
```

Comment:
This was dumb of me not to note earlier, but this project isn't being transpiled atm. This should be replaced w/ something not using Array.prototype.find and without the arrow function.

Or we could investigate how much time transpiling w/ buble would add, I'm not opposed to it.